src/log/log.c

```diff
@@ -673,6 +673,10 @@ __log_prealloc(WT_SESSION_IMPL *session, WT_FH *fh)
 		return (__log_zero(session, fh,
 		    WT_LOG_END_HEADER, conn->log_file_max));
 
+	/* If configured to not extend the file, we're done. */
+	if (conn->log_extend_len == -1)
```

Comment:
This may change of you decide to use 0. Also consider adding for data files too - or does that fall out if someone uses 0.

Reply:
If you agree with my comment above, then we would use -1 to disable log file extention.